components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -276,13 +276,9 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             );
         }
         if (!!result.spendingLimitReachedOnCostCenter) {
-            throw new ResponseError(
-                ErrorCodes.PAYMENT_SPENDING_LIMIT_REACHED,
-                "Increase spending limit and try again.",
-                {
-                    attributionId: result.spendingLimitReachedOnCostCenter,
-                },
-            );
+            throw new ResponseError(ErrorCodes.PAYMENT_SPENDING_LIMIT_REACHED, "Increase usage limit and try again.", {
+                attributionId: result.spendingLimitReachedOnCostCenter,
```

Comment:
Do you want to rename the `spendingLimitReachedOnCostCenter` field too?

Reply:
Good question. I wasn't sure if we want to go back eventually, therefore tried to not change API.

@jldec, can we assume that #12637 is a temporary issue, and we'll change semantics eventually? If not, then the clean-up should include API field as well.

Reply:
I would consider it a permanent change - thanks @AlexTugarev

Reply:
@andrew-farries, I changed the API related stuff, but left out `d_b_costcenter.spendingLimit`, is it ok with you?

Reply:
yes